packages/components/pages/f-contact-preferences/src/store/_tests/contactPreferences.module.test.js

```diff
@@ -0,0 +1,149 @@
+import contactPreferencesModule from '../contactPreferences.module';
+import {
+    UPDATE_PREFERENCE,
+    UPDATE_PREFERENCES
+} from '../../constants';
+import {
+    contactPreferencesViewModel,
+    token,
+    contactPreferencesGetResponse,
+    contactPreferencesUpdateModel
+} from '../../../test-utils/setup';
+
+describe('ContactPreferences Store', () => {
+    let httpClientMock;
+    let getPreferencesMock;
+    let postPreferencesMock;
+
+    beforeEach(() => {
+        // Arrange
+        getPreferencesMock = jest.fn(() => contactPreferencesGetResponse);
+        postPreferencesMock = jest.fn(() => {});
+        httpClientMock = {
+            getPreferences: getPreferencesMock,
+            postPreferences: postPreferencesMock
+        };
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should create default state when initialised.', () => {
+        // Assert
+        const actualState = contactPreferencesModule.state();
+
+        expect(actualState).toEqual({
+            preferenceVersionViewed: 0,
+            preferences: []
+        });
+    });
+
+    describe('actions ::', () => {
+        describe('loadPreferences ::', () => {
+            it('should call the getPreferences api method with the correct parameters', async () => {
+                // Act
+                await contactPreferencesModule.actions.loadPreferences({ commit: jest.fn() }, { api: httpClientMock, authToken: token });
+
+                // Assert
+                expect(getPreferencesMock).toHaveBeenCalledWith(token);
+            });
+
+            it(`should call ${UPDATE_PREFERENCES} mutation with the correct data`, async () => {
+                // Arrange
+                const commitSpy = jest.fn();
+                const expectedVersion = contactPreferencesViewModel.preferenceVersionViewed;
+                const expectedPreferences = contactPreferencesViewModel.preferences.filter(e => !!e.visible);
```

Comment:
Is this bit necessary?
```suggestion
                const expectedPreferences = contactPreferencesViewModel.preferences.filter(e => e.visible);
```